src/audio/module_adapter/module_adapter.c

```diff
@@ -736,7 +736,8 @@ static int module_adapter_simple_copy(struct comp_dev *dev)
 		src_c = attr_container_of(mod->input_buffers[i].data,
 					  struct comp_buffer __sparse_cache,
 					  stream, __sparse_cache);
-		comp_update_buffer_consume(src_c, mod->input_buffers[i].consumed);
+		if (!mod->skip_src_read_pointer_update)
```

Comment:
can you inline comment here and below on what condition would activate the update ?